### ISSUE TITLE
Possible Security Issue

### ISSUE THREAD

**Issue by arnavb** [OpenGenus/memes#53]
The [auth.ini](https://github.com/OpenGenus/memes/blob/master/auth.ini) file in the root directory contains credentials for social media sites. This is obviously not a recommended practice, because others can freely use your API key to access the website. It is recommended that secrets are NEVER committed to version control.

For maximum security, I would recommend rewriting the git history so that the secrets file isn't there at all.

**Comment by adityaprakash-bobby** [comment]
Yes, that will be removed before the first release of the project.
